fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -59,6 +59,7 @@
  */
 public class FHIRResource {
     private static final Logger log = java.util.logging.Logger.getLogger(FHIRResource.class.getName());
+    protected static final boolean NO_SKIP_VALIDATION = false; // do not skip validation
```

Comment:
small comment, no action necessary, just awareness. 
it comes off as a double negative and makes it slightly less readable

Reply:
FWIW I find `DO_VALIDATION = false` less mind-numbing than `NO_SKIP_VALIDATION = false`

Reply:
I've addressed this by taking John's suggestion of using a couple of default methods in the interface.